fe/fe-core/src/main/java/org/apache/doris/nereids/rules/RuleSet.java

```diff
@@ -37,6 +38,7 @@
 public class RuleSet {
     public static final List<Rule<Plan>> ANALYSIS_RULES = planRuleFactories()
             .add(new BindRelation())
+            .add(new AggregateRewrite())
```

Comment:
maybe it is better to use two rule set, one for bind, another for rewrite

Reply:
why